lib/cocina/models/builders/title_builder.rb

```diff
@@ -63,17 +65,18 @@ def initialize(strategy:, add_punctuation:, only_one_parallel_value: true)
           @only_one_parallel_value = only_one_parallel_value
         end
 
-        # @param [[Array<Cocina::Models::Title>] cocina_titles the titles to consider
+        # @param [Array<Cocina::Models::Title>] cocina_titles the titles to consider
+        # @param [String, nil] part_label the partLabel to add to the title for digital serials
         # @return [String, Array] the title value for Solr - for :first strategy, a string; for :all strategy, an array
         #   (e.g. title displayed in blacklight search results vs boosting values for search result rankings)
         #
         # rubocop:disable Metrics/PerceivedComplexity
-        def build(cocina_titles)
+        def build(cocina_titles, part_label: nil)
```

Comment:
We could consider injecting either the `part_label` or the `catalog_links` in the `.new` call on line 26 and making it an `attr_reader` on the instance to avoid having to pass it around and default it in a couple places, but I don't think that's a biggie.

Reply:
Took this approach, thanks for the suggestion!